server-master/src/main/java/com/aliyun/emr/rss/service/deploy/master/MasterUtil.java

```diff
@@ -17,133 +17,474 @@
 
 package com.aliyun.emr.rss.service.deploy.master;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
-import java.util.Random;
+import java.util.*;
```

Comment:
Refactor this file to be more readable

Reply:
Import more than 10 classes from the same package.